run.sh

```diff
@@ -21,6 +23,8 @@ rebuild_all() {
     -t hardhat-tutorial-nft \
     -t hardhat-tutorial-precompiled-token \
     -t hardhat-tutorial-dex \
+    -t hardhat-tutorial-upgradeable-greeter \
+    -t advanced-escrow \
```

Comment:
@shunjizhan have you tried running it in the CI? It forces block generation within the tests, so it might be a bit tricky 🤔

Reply:
I ran it locally and it worked without the loop helper, maybe it was fixed by this? https://github.com/AcalaNetwork/bodhi.js/pull/436/files#diff-8e05d01e1b336d3501a54605915fd1c9116c51bc23417352c54359c4ca197119L1386

Reply:
oh wait I think `yarn test-mandala` auto run the loop helper, so should be fine